docs/getting-started/install-feast/README.md

```diff
@@ -12,3 +12,15 @@ This guide installs Feast into an AWS environment using Terraform. The Terraform
 
 {% page-ref page="kubernetes-amazon-eks-with-terraform.md" %}
 
+### Azure AKS \(with Terraform\)
+
+This guide installs Feast into an Azure environment using Terraform. The Terraform script is opinionated and intended to allow you to start quickly.
+
+{% page-ref page="kubernetes-azure-aks-with-terraform.md" %}
+
+### Google Cloud GKE \(with Terraform\)
+
+This guide installs Feast into a Google Cloud environment using Terraform. The Terraform script is opinionated and intended to allow you to start quickly.
```

Comment:
Should this be here? It's GKE?

Reply:
yes it should've been added to this page before so I just added it in as part of this diff